src/screens/VisualTests/VisualTests.stories.tsx

```diff
@@ -341,8 +362,11 @@ export const RunningBuildInProgress: Story = {
 };
 
 export const Pending: Story = {
+  args: {
+    // @ts-expect-error
+    AddonVisualTestsBuild: constructResult({ storyBuild: pendingBuild }),
```

Comment:
For some reason it errors if I set this arg on the `Story` but not on the `Meta`.

Reply:
Does it make more sense to make more sense for the arg to be the input to `constructResult` if we want the user to be able to edit this in the controls panel?

Reply:
How would that work @shilman? `constructResult` is specific to this query...

Reply:
Ah, yes, if you use `typeof meta`, it only looks at the component type, and it can really do anything with the satisfies part.

You will have to do something as:

```ts
type StoryArgs = Parameters<typeof VisualTests>[0] & {
    AddonVisualTestsBuild: ResultOf<typeof QueryBuild>;
}

const meta = { } satisfies Meta<StoryArgs>;

type Story = StoryObj<StoryArgs>
```

Reply:
That is right, you have to set them optional yourself in StoryObj.

I have thought about this scenario, but I think there is no way, as the `satisfies` constraint has no effect on the inferred return type of `meta`. 

Factories would solve this problem 🤓

Reply:
I figured you'd say that :)

Reply:
This worked OK, FTR: 
![image](https://github.com/chromaui/addon-visual-tests/assets/132554/1ab7c29c-8e3b-4e59-95e6-612f2e6443c1)

Only problem was a couple of stories that had a `render()` function, the `args` were typed wrongly :/